tests/e2e/loadbalancer.go

```diff
@@ -132,6 +132,15 @@ var _ = Describe("[cloud-provider-aws-e2e] loadbalancer", func() {
 				framework.ExpectNoError(getLBTargetCount(context.TODO(), lbDNS, clusterNodesCount), "AWS LB target count validation failed")
 			},
 		},
+		{
+			Name:           "NLB should configure with security groups",
+			ResourceSuffix: "nlb-sg",
+			Annotations: map[string]string{
+				annotationLBType: "nlb",
+			},
+			// TODO find a way to change the cloud-config to NLBSecurityGroupMode=NLBSecurityGroupModeManaged
+			// The configuration must be restored when the test finished.
```

Comment:
This is a TODO and need help: I really didn't find a way yet to change/patch the cloud-config in the running cluster to run a single scenario. Thoughts?

cc @elmiko